templates/example_cell_clustering.ipynb

```diff
@@ -26,10 +26,16 @@
     "import subprocess\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    cell_table_name = 'cell_table_size_normalized.csv'
</pre><p>This can be combined with the cell above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/460/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>